src/theme/Toggle/index.js

```diff
@@ -1,77 +0,0 @@
-/**
```

Comment:
Were these not being customized in spite of having been swizzled?

Reply:
The component was swizzled solely to customize the appearance via CSS. See from line 107 of the also deleted stylesheet [src/theme/Toggle/styles.module.css](https://github.com/gruntwork-io/docs/blob/f32a0783814c57feab50ea4746eab50a034ea0c2/src/theme/Toggle/styles.module.css#L107).

I showed @ebeneliason the new toggle in this version of Docusaurus and we agreed that there is no need to swizzle anymore as the new appearance is in line with what we implemented.